input/commands.go

```diff
@@ -79,9 +79,9 @@ func cursorCommands() []Command {
 			},
 		},
 		{
-			Name: "cursor down (down arrow or j)",
+			Name: "cursor down (down arrow, j or Enter)",
 			BuildExpr: func() vm.Expr {
-				return verbCountThenExpr(altExpr(keyExpr(tcell.KeyDown), runeExpr('j')))
+				return verbCountThenExpr(altExpr(keyExpr(tcell.KeyDown), runeExpr('j'), keyExpr(tcell.KeyEnter)))
```

Comment:
this change looks unrelated to W/E/B commands. Could you open a separate PR for this?

I think the behavior of the "enter" key might be slightly different than this, since in vim it seems to moves the cursor to the beginning of the line.

Reply:
Will do.

Reply:
Done